tests/azure-test/azure_test_suite_test.go

```diff
@@ -0,0 +1,36 @@
+package azuretest
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"testing"
+
+	testutils "k8s.io/cloud-provider-azure/tests/azure-test/utils"
+
+	"github.com/golang/glog"
+	. "github.com/onsi/ginkgo"
```

Comment:
why there is `.` here?

Reply:
It refers to the ginkgo's official document: http://onsi.github.io/ginkgo/
By this, we use namespace ginkgo